CONTRIBUTORS.md

```diff
@@ -56,6 +56,62 @@
 - [is343](https://github.com/is343)
 - [Meet Pandya](https://github.com/meet-k-pandya)
 - [Peter Spenler](https://github.com/peterspenler)
+ - [JoshuaBoniface](https://github.com/joshuaboniface)
```

Comment:
Unrelated changes due to a wrong merge conflict resolution.

Reply:
Yeah... the automerging seemed to do some weird things when rebasing... I've fixed it now and squashed the commits